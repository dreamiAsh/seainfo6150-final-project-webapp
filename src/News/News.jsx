import React from 'react'
import Styles from "./News.module.css"
import NewsCom from "../Components/NewsCom/NewsCom"

const News = () => {
    return (
        <div className={Styles.box}>
            <NewsCom/>
        </div>
    )
}

export default News
