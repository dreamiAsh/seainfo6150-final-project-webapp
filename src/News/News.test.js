import React from "react";
import { render, fireEvent } from "@testing-library/react";
import News from "./News.jsx";

it("renders the News component correctly", () => {
  const { container } = render(<News />);
  expect(container).toMatchSnapshot();
});