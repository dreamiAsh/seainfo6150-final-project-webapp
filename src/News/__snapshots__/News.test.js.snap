// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the News component correctly 1`] = `
<div>
  <div
    class="box"
  >
    <div
      class="box"
    >
      <ul />
    </div>
  </div>
</div>
`;
